epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+# Security and Privacy Questionnaire for EPUB 3.3
+
+Written by Dave Cramer (invited expert) and the EPUB 3 Working Group, April 29, 2021.
+
+## Links to specifications
+
+* [EPUB 3.3 Overview](https://w3c.github.io/epub-specs/epub33/overview/), a non-normative overview of EPUB 3.3
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+
```

Comment:
Maybe worth indicating which documents are rec-track.

Reply:
I have made that change. By the way, how would one tell that the A11Y techniques doc is not rev-track from looking at the document itself?

Reply:
On the editor's draft? It is well hidden: there is a sentence in the status section which says "The group does not expect this document to become a W3C Recommendation.". It is generated by respec.